src/App.css

```diff
@@ -14,23 +14,23 @@
   }
 }
 
-.App-header {
-  background-color: #2db44b;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
+/* .App-header { */
```

Comment:
Please make sure to remove code that is not being used rather than just commenting it out.

Reply:
Okay, i had only concetrated on the home page but am going to be looking into all what you have said this week.Thanks